example/disasm/full.py

```diff
@@ -24,296 +22,340 @@
+                                                     ",".join(Machine.available_machine()))
+    parser.add_argument('-f', "--followcall", action="store_true",
+                        help="Follow call instructions")
+    parser.add_argument('-b', "--blockwatchdog", default=None, type=int,
+                        help="Maximum number of basic block to disassemble")
+    parser.add_argument('-n', "--funcswatchdog", default=None, type=int,
+                        help="Maximum number of function to disassemble")
+    parser.add_argument('-r', "--recurfunctions", action="store_true",
+                        help="Disassemble founded functions")
+    parser.add_argument('-v', "--verbose", action="count", help="Verbose mode",
+                        default=0)
+    parser.add_argument('-g', "--gen_ir", action="store_true",
+                        help="Compute the intermediate representation")
+    parser.add_argument('-z', "--dis-nulstart-block", action="store_true",
+                        help="Do not disassemble NULL starting block")
+    parser.add_argument('-l', "--dontdis-retcall", action="store_true",
+                        help="If set, disassemble only call destinations")
+    parser.add_argument('-s', "--simplify", action="count",
+                        help="Apply simplifications rules (liveness, graph simplification, ...)",
+                        default=0)
+    parser.add_argument("--base-address", default=0,
+                        type=lambda x: int(x, 0),
+                        help="Base address of the input binary")
+    parser.add_argument('-a', "--try-disasm-all", action="store_true",
+                        help="Try to disassemble the whole binary")
+    parser.add_argument('-i', "--image", action="store_true",
+                        help="Display image representation of disasm")
+    parser.add_argument('-c', "--rawbinary", default=False, action="store_true",
+                        help="Don't interpret input as ELF/PE/...")
+    parser.add_argument('-d', "--defuse", action="store_true",
+                        help="Dump the def-use graph in file 'defuse.dot'."
+                             "The defuse is dumped after simplifications if -s option is specified")
+    parser.add_argument('-p', "--ssa", action="store_true",
+                        help="Generate the ssa form in  'ssa.dot'.")
+    parser.add_argument('-x', "--propagexpr", action="store_true",
+                        help="Do Expression propagation.")
+    parser.add_argument('-e', "--loadint", action="store_true",
+                        help="Load integers from binary in fixed memory lookup.")
+    parser.add_argument('-j', "--calldontmodstack", action="store_true",
+                        help="Consider stack high is not modified in subcalls")
+
+    args = parser.parse_args()
+
+    full(args.filename, verbose=args.verbose, raw_binary=args.rawbinary, base_address=args.base_address,
```

Comment:
Why don't you pass directly 'args' to full instead of a making a big 20 arguments function?